service-api/module/Application/src/Nino/Validator.php

```diff
@@ -20,6 +20,9 @@ public function __construct(
     //@TODO implement when we have API access
     public function validateNINO(string $nino): string
     {
-        return 'Pass';
+        $validNinos = [
+            'AA112233A'
+        ];
+        return in_array(str_replace(' ', '', $nino), $validNinos) ? 'Pass' : 'Fail';
```

Comment:
This validator is what will be used for the real integration, so shouldn't have any logic in it yet. For now we should be using `Application\Mock\Nino\Validator` instead of this "real" file